pkg/csi/service/common/topology.go

```diff
@@ -163,11 +163,11 @@ func GetHostsForSegment(ctx context.Context, topoSegment map[string]string, vCen
 				}
 				for _, childType := range folder.ChildType {
 					switch childType {
-					case "vim.Datacenter":
```

Comment:
I see you are changing all childtypes. how test passed when we introduced this change? Did we miss any specific testing?

Reply:
Topology 2 level testbed did not have any DC or ComputeResource under Folders, that is why this check passed.